Dockerfile

```diff
@@ -39,4 +39,8 @@ RUN echo "deb [trusted=yes] https://apt.secrethub.io stable main" > /etc/apt/sou
 	apt-get update && \
 	apt-get install -y secrethub-cli
 
+WORKDIR /home/secrethub-dotnet/secrethub-xgo/
+COPY secrethub-xgo/ ./
+RUN go mod download -x
+
```

Comment:
These lines copy the code into the container and fetch it's dependencies, however since this is done in the `Dockerfile` these will be stored in the container's image. So if we change the dependencies or the code the changes will not be reflected in the container (only if you rebuild it).

To avoid this issue I suggest we find out where `go build` the dependencies it fetches and mount that directory as another volume in the container.

Reply:
Will look into it then.

Reply:
I'll take over this PR